controllers/admin/AdminProductsController.php

```diff
@@ -1477,9 +1477,7 @@ public function ajaxProcessUpdateImagePosition()
             $json = stripslashes($json);
             $images = json_decode($json, true);
             foreach ($images as $id => $position) {
-                $img = new Image((int) $id);
-                $img->position = (int) $position;
-                $res &= $img->update();
+                $res &= Db::getInstance()->execute('UPDATE ' . _DB_PREFIX_ . 'image SET position = ' . $position . ' WHERE id_image = ' . $id);
```

Comment:
Is this problem still exists in the latest release?

Reply:
I have no idea, this was for 1.6, even though the bug was there in 1.7.6.1 (https://github.com/PrestaShop/PrestaShop/issues/12832#issuecomment-526241845)

I haven't been working on PrestaShop for over a year now so I'm totally out of the loop

Reply:
I will try to investigate to see if the problem still exists in 1.7, and if your fix is the right one. Thanks for your quick reply